Android/Java/app/src/main/java/com/ironsource/ironsourcesdkdemo/DemoActivity.java

```diff
@@ -119,50 +144,68 @@ public void showRewardedVideoButtonTapped(View view) {
 
     public void loadInterstitialButtonTapped(View view) {
         // This will load an Interstitial ad
-
         log("loadInterstitial");
-        IronSource.loadInterstitial();
+        if (mInterstitialAd != null) {
+            mInterstitialAd.loadAd();
+        }
```

Comment:
Do we want to log a message when this case isn't happed? @ronala

Reply:
@ArielBarsheshetIS  it should not occur according to the flow. Only when the init success occurs, the load button is enabled that allows this flow to start in which case the mInterstitialAd is not null. Adding null check is simply good practice

Reply:
Got it, agree with you :))